build/upgrade_pdfjs.sh

```diff
@@ -56,13 +55,11 @@ sed -e 's@;r.setFlags(o.AnnotationFlag.HIDDEN)@@' -i '' ${DOC_STATIC_ASSETS_PATH
 # This fix may not need to be applied if next upgrade is >= 2.5.x
 # See https://github.com/box/box-content-preview/pull/1414 for more details
 
-# Minify using Google Closure Compiler, options:
-# Output to ES5 (Box supports Chrome, Edge, IE11, Firefox, Safari, and newer versions of iOS, Android)
-# Do not minify pdf.js or pdf.worker.js, as the closure compiler will mangle function names and cause bugs
+# Minify using Babels
```

Comment:
`Babels` -> `babel-minify`?

Reply:
Oh actually, this is just a babel command, but I should fix the typo; I ran into some issues using the babel-minify command aha...